.pipelines/ESRPSigning_core.json

```diff
@@ -77,6 +77,10 @@
                 "modules\\launcher\\Wox.dll",
                 "modules\\launcher\\Wox.Infrastructure.dll",
                 "modules\\launcher\\Wox.Plugin.dll",
+                "modules\\launcher\\e_sqlite3.dll",
```

Comment:
These need to be in the bottom set, not top set of signing. These should be treated as 3rd party